### ISSUE TITLE
[UBS Admin Cabinet] Order statuses

### ISSUE THREAD

**Issue by Mariana-Khishchenko** [ita-social-projects/GreenCity#2538]
As an administrator, I want to be able to change order statuses so that I can organize and manage orders.

**Assumptions & Constraints**
1. Only administrators with set rights are able to change statuses of orders

**Preconditions**
1. User is logged in as administrator.
2. User navigates to the Orders tab

**Business rules**
1. Only administrators with access should be able to view Admin Cabinet
2. When changing statuses - statuses are enabled in order of succession. e.g. According to the state diagram. Jumping over from ‘Сформовано' to 'На маршруті' is prohibited. All previous steps and data must be entered manually and\or automatically

### Acceptance Criteria

Supported statuses: | Automation | Requirements | Comments
-- | -- | -- | --
Сформовано | Auto | Is assigned to order after order was submitted by the user (On 3 stage of form submission) | Order submitting is described in the story ‘[UBS] Fill in and submit Order form’
Оплачено | Auto | Is assigned to order after the order payment was received (response from the payment system or added manually by admin). May be assigned to order after status ‘Підтверджено’ - if the user did not confirm the date of pick-up | Confirmation is received, and ‘Оплата’ Column from ‘[Admin Cabinet] Current orders view’ story has a payment ID from the payment system or from the form that was manually filled in by the admin
Узгодження | Manual | Set after admin created user delivery-list. Prior to confirmation of pick-up |  
Підтверджено | Manual | Is assigned after client confirmation of due date and time. Status is shown after status ‘Оплачено’ and ‘Не вивезли’ | Due date is confirmed and ‘Дата вивезення’ with ‘Час вивезення’ Columns from ‘[Admin Cabinet] Current orders view’ are filled 
На маршруті | Auto | Is auto-assigned in the delivery day at 00:00. |  
Не вивезли | Manual | Status is shown only after status ‘На маршруті’ was set. When changing to this status - a pop-up window is shown with the comment section and ‘add photo’ control. Photo addition is not a requirement to change to this status. | After inspection of packaged litter - courier confirms that it was not properly packaged. It wasn`t possible for the client to eliminate the packaging issue
Виконано | Manual | Status is shown only after status ‘На маршруті’ was set. |  
Скасовано | Manual | Status may be changed from all statuses except ‘Виконано’. When changing to this status - a pop-up window is shown with the comment section | User must be able to cancel the order only from Сформовано stage. Admin must be able to cancel the order from all statuses except Виконано
Виконано/Не оплачено | Manual | Status is shown only after status ‘На маршруті’ was set. |  

![image](https://user-images.githubusercontent.com/73824853/114718171-6bf9d380-9d3e-11eb-9977-ccea5036ed05.png)

**Out of scope**
Order Cancellation

**Epic link**
E.g.: Epic #2195 

**Labels to be added**
"User story", Correspondind Functionality (e.g. User), Priority ("pri: "), "Corresponding Epic"

### Tasks 
1. - [ ] sample task.
